### PR TITLE
bug(md-select): change the arrow position to absolute

### LINKED ISSUE #3112: bug(md-select) - not rendering correctly inside a flex layout (pure css) when having ngModel
#### Bug, feature request, or proposal:
bug

#### What is the expected behavior?
When inside a flex layout (pure css), the md-select must behave consistently, independent if it has a ngModel or not.

#### What is the current behavior?
When the md-select has a ngModel, and it's width is different from the default at page load (inside of a flex layout), the component does not fit inside the container.

This plunker demostrate this behavior:
[Plunker](http://plnkr.co/edit/O1V6N8jCyWpHdu1VTB12?p=preview)

#### What is the use-case or motivation for changing an existing behavior?
We need to display our pages in a responsive layout and are using just pure css flex layout while we wait for [Angular Flex Layout](https://github.com/angular/flex-layout) to became more stable.

#### Which versions of Angular, Material, OS, browsers are affected?
@angular/material: 2.0.0-beta.1
Firefox: 51.0.1 (64-bit)
Chromium: 53.0.2785.143
Chrome: 56.0.2924.87

### PR DESCRIPTION
**Pull request by jefersonestevo**
Fixes #3112

### CODE REVIEW

File: src/demo-app/select/select-demo.html [hunk 1]
```diff
@@ -54,3 +54,27 @@
 
 </div>
 <div style="height: 500px">This div is for testing scrolled selects.</div>
+
+<md-card>
```
Comment (crisbeto) [inline src/demo-app/select/select-demo.html]: I don't think we need this example. The examples in the demo app are there to be able to click around the main use cases, they're not for actual testing.

**Comment by jefersonestevo** [comment]
@crisbeto conflicts resolved and I've removed the demo for flex layout from the demo page. Anything else that needs to change here?

**Comment by crisbeto** [comment]
Not code-wise, but you have a few merge conflicts which can cause issues with our scripts. Can you squash all your commits into one?

**Comment by jefersonestevo** [comment]
@crisbeto As these changes are really small and there are some merges with lots of commits here, I've created a new pull request with just the changes based on the current master. I'm closing this one and we can continue at #3580

**Comment by crisbeto** [comment]
Well, it's already done, but FYI you can still squash commits after the PR has been opened.

**Pull request closed without merging**